backend/api/Services/Models/IsarMissionDefinition.cs

```diff
@@ -72,8 +72,8 @@ public struct IsarInspectionDefinition
         [JsonPropertyName("inspection_target")]
         public IsarPosition InspectionTarget { get; set; }
 
-        [JsonPropertyName("analysis_types")]
-        public string? AnalysisTypes { get; set; }
+        [JsonPropertyName("analysis_type")]
+        public string AnalysisType { get; set; }
```

Comment:
Why is this no longer optional?

Reply:
Ah because AnalysisType (enum) was made mandatory before but not it is not so should be fixed.

Reply:
Made optional again. Had it removed earlier because I planned to introduce a seperate analysistype called default but that is no longer the case so should be optional now.